### DISCUSSION TITLE
Easier access to switch spaces and spaces rename

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#870] [Ideas]
### What feature would you like to see?

Hi, would it be possible to get an option to change the switching of spaces to icons so it goes from two clicks to one click? Something like how ARC does it, or something else if you have something better in mind -- ease of access is ideal. This could be an option toggle in the settings or something for the current method or this method. I'm not sure if this is in your vision for the browser, just putting this out there.
![image](https://github.com/user-attachments/assets/af26b17b-6e97-4dda-af2d-c30be828f64a)

Also, any option to rename default space/delete default space? I like to have a home and a work space, but this is seems to be the limit I could change it to.
![image](https://github.com/user-attachments/assets/cc163605-0fa0-40c7-9c68-f8197a54cc13)

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/260_

**Comment by Hate9** [comment]
This would be excellent. I'd love it if it worked like it does in Opera and Floorp, with the spaces listed on a scrollable sidebar. This feature is actually the only feature keeping me from switching to Zen from Floorp.

Alternatively, I'd also be fine with it working more like Simple Tab Groups (at least when it's used as a sidebar for vertical tabs), where you can click on the top to switch your tabs list to a list of your workspaces.

Either is fine, but the current option is definitely *very* lacking.

**Reply by ChristinWhite** [reply]
After a couple of years using SigmaOS and Sidebery I can't imagine using a browser that doesn't have a vertical workspaces sidebar. I keep a lot of projects grouped in my workspaces and horizontal implementations don't offer enough space for me.

**Comment by FabiKel** [comment]
As I wrote in the original issue: 

> that would be great. I use Sideberry and have the spaces at the top too. I think it is wasted _space_ to have a single big button to switch between workspaces.

![358242453-74103073-5dd2-401b-ab08-45a882eba736](https://github.com/user-attachments/assets/862bd11f-3fb5-459a-a3fb-bddaeda5eed3)

Functions that I use daily or frequently are
- the tab search, which is located at the same place in Sideberry
- moving tabs between workspaces via drag and drop
- unload all tabs of a workspace with a mouse click
- hide the workspace, useful for rarely used workspaces
- cool gimmick: display of existing tabs (like in the picture)

**Comment by kristijanribaric** [comment]
Implemented since `1.0.1-a.7`
Make sure to go to `about:preferences#zenTabManagement` page and make sure you have this option checked
![image](https://github.com/user-attachments/assets/db508406-a926-4040-9c3b-e9eefb681a0b)
